src/components/state-badge.component.js

```diff
@@ -58,19 +47,25 @@ export const StateBadge = ({
     issueText = translate('issue.main.states.closed', locale);
   }
 
-  let issueStyle = {};
+  let issueStateColor;
 
-  if (issueState === 'merged') {
-    issueStyle = styles.mergedIssue;
-  } else if (issueState === 'open') {
-    issueStyle = styles.openIssue;
-  } else if (issueState === 'closed') {
-    issueStyle = styles.closedIssue;
+  switch (issueState) {
+    case 'merged':
+      issueStateColor = colors.purple;
+      break;
+    case 'open':
+      issueStateColor = colors.green;
+      break;
+    case 'closed':
+      issueStateColor = colors.red;
+      break;
+    default:
+      issueStateColor = colors.gray;
   }
 
   return (
-    <View style={[styles.badge, style, issueStyle]}>
-      <Text style={styles.text}>{issueText}</Text>
-    </View>
+    <Badge color={issueStateColor} style={style}>
```

Comment:
You are not passing `style` anymore from `issue-description.component.js`, but you're still using the prop here .. Could you give it a second look?

Reply:
Yeah I thought to get completely rid of the style prop but it could still be useful in case there is a need to override something from a parent component without modifying again this one instead.

But it's up for discussion, don't know if it is really needed or not so I left it there 😆

Reply:
Prop `style` is also used in `repository-section-title.component.js`, which is a useless component. I remembered @housseindjirdeh confirmed that in somewhere, but I forgot the detail issue/pr number. I will collect feedbacks in #725.

Reply:
it was on Gitter, I added the relevant screenshot in the issue you opened.